README.md

```diff
@@ -1,6 +1,12 @@
 # cmp-nvim-lsp
 
-nvim-cmp source for neovim builtin LSP client
+nvim-cmp source for neovim's built-in language server client.
+
+# Capabilities
+
+Language servers provide different completion results depending on the capabilities of the client. Neovim's default omnifunc has basic support for serving completion candidates. nvim-cmp supports more types of completion candidates, so users must override the capabilities sent to the server such that it can provide these candidates during a completion request. These capabilities are provided via the helper function `require('cmp_nvim_lsp').update_capabilities` 
+
+As these candidates are sent on each request, **adding these capabilities will break built-in the built-in omnifunc support for neovim's language server client**. `nvim-cmp` provides manually triggered completion that can replace omnifunc. See `:help cmp-faq` for more details.
```

Comment:
`will break built-in the built-in omnifunc` seems like duplicated `build-in.

Reply:
whooopppsss